topologies/binding/ssh_test.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2022 Google LLC
```

Comment:
The license is outdated. Can you use addlicense tool?

https://github.com/google/addlicense